WinUI3/AK.Toolkit.WinUI3.AutoCompleteTextBox/AutoCompleteTextBox.cs

```diff
@@ -142,10 +143,11 @@ protected override void OnApplyTemplate()
     {
         base.OnApplyTemplate();
 
-        if (Resources.TryGetValue("extControlPlaceholderForeground", out var value) is true)
```

Comment:
I guess this missing "T" was causing the problem.

Reply:
no, i tested with full name `TextControlPlaceholderForeground` and The result was the same as before.

Reply:
Still weird but your change definitely works! Thanks!  😊